packages/frontend/src/components/common/PivotTable/TitleCell.tsx

```diff
@@ -2,14 +2,12 @@ import { TitleFieldValue } from '@lightdash/common';
 import { Box } from '@mantine/core';
 import { FC } from 'react';
 
-type TitleCellProps = {
+interface TitleCellProps {
```

Comment:
I saw a few `type` -> `interface` changes on your previous PR, what's the benefit of switching? We use `type` in most of the codebase

Reply:
We should stick to one. Usually we default to types since they allow us to use the pattern `type ABC = {type: A} | {type: B} | ...` - like in `ResourceViewItem`. Otherwise there is not much benefit to the interface, unless we are doing some OOP

Reply:
But let's not block the PR this time!